package.json

```diff
@@ -19,7 +19,7 @@
     "node": ">=0.8"
   },
   "dependencies": {
-    "backbone": "1.1.2",
+    "backbone": "1.2.3",
```

Comment:
Yeah, I haven't used it in a long time (feels like years). I'd rather not support it anymore since there are better options (NPM).